queue/queue.py

```diff
@@ -1,15 +1,25 @@
 class Queue:
-  def __init__(self):
-    self.size = 0
-    # what data structure should we
-    # use to store queue elements?
-    self.storage = 
-
-  def enqueue(self, item):
-    pass
-  
-  def dequeue(self):
-    pass
-
-  def len(self):
-    pass
+    def __init__(self):
+        self.size = 0
+        # what data structure should we
+        # use to store queue elements?
+        self.storage = []
```

Comment:
I would strongly consider using a LinkedList() as the structure of choice for storage in a Queue class.  Using a list/array does not give us constant time operations with both adding/removing items from the queue.